.gitignore

```
@@ -1,3 +1,4 @@
 pkg/
 src/
 *.pkg.tar.xz
```

```suggestion
*.tar.xz
```
might as well shorten that one as well